app/(auth)/sign-up/page.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import SignUpForm from "@/components/auth/SignUpForm";
+import { SocialAuthGroup } from "@/components/auth/SocialAuthGroup";
+const SignUpPage = async () => {
+  return (
+    <div className="flex-center flex-col">
+      <span className="text-left text-2xl leading-8 tracking-[-0.02rem] text-white-100">
+        Create An Account
+      </span>
+      <div className="h-[472px] w-[400px]">
+        <SignUpForm />
+        <SocialAuthGroup />
+      </div>
+    </div>
```

Comment:
```suggestion
    <section>
      <span className="display-2-bold text-white-100">
        Create An Account
      </span>
      <SignUpForm />
      <SocialAuthGroup />
    </section>
```

Because we've changed out layout to control our layout- we don't need each page to handle it when they aren't going to change between route groups. 

We can remove a div and now use a 'section' tag to indicate that this is a specific section of the page.

We dont' need to align the text to the left, as now it's not in a centered flexbox. So it will already be left aligned.

We can also use the proper "display-2-bold" class, which is easy to find as when clicking the text in figma, it will tell us exactly which one to use :)

![image](https://github.com/agordon123/git_note/assets/4108484/41777ad2-7661-42cc-b79b-36c79d7d848f)